### ISSUE TITLE
Use changesets

### ISSUE THREAD

**Issue by rufuspollock** [flowershow/flowershow#16]
Set up https://github.com/changesets/changesets for this repo and briefly document it into the developer workflow

Good recent setup instructions https://dnlytras.com/blog/using-changesets/

## Acceptance

* [ ] We are using changesets to track changes e.g. one changeset example
* [ ] Automating doing releases and pushing to npm via github actions (?)
  * [ ] register @flowershow org on npm
  * [ ] github action
* [ ] short docs on our use of changests in developer docs e.g. https://flowershow.app/docs/developers

## Tasks

* [x] Quick research on how to do this
* [x] Recommend whether to proceed (e.g. are benefits worth the costs)
* [ ] Next steps ...
* [ ] ...

**Comment by rufuspollock** [comment]
@abhishekgahlot could you summarize your quick research on how to do this e.g what changesets are, why useful, how to use etc.

**Comment by abhishekgahlot** [comment]
Changesets are way to manage your versioning and changelogs. Sometime we end up grouping changes in one PR. Using changesets we can group multiple changes and create a `release` , `minor` or major patch.

More info on that here: https://github.com/changesets/changesets/blob/main/docs/detailed-explanation.md

The PR associated with is: #142
